### ISSUE TITLE
Initialize from Array does not work

### ISSUE THREAD

**Issue by Bjvanminnen** [RakanNimer/react-google-charts#31]
https://github.com/RakanNimer/react-google-charts/blob/master/src/components/Chart.js#L91
```
drawChart: function() {

		if ((this.props.data !== null && this.props.data.length === 0) || this.props.columns.length === 0) {
			return;
		}
...
```

The above code means that if we don't specify a columns prop, drawChart exits without doing anything. From what I can tell, this means that the first example in your README where you provide options and data, but no columns, does not display anything.

**Comment by dmmalam** [comment]
+1

**Comment by RakanNimer** [comment]
Yep, absolutely right. 
This was fixed in the latest version on npm : 0.2.1.

Cheers !
